robotd/devices.py

```diff
@@ -227,10 +227,16 @@ def start(self):
         self.vision_thread = Thread(target=self._vision_thread)
         self.vision_thread.start()
 
+    @staticmethod
+    def _serialise_marker(marker : Token):
+        d = marker.__dict__
```

Comment:
use `dict(marker)` over `__dict__`. They do the same things, it's just 1 is far nicer!

Reply:
If `Token` is a `namedtuple` I think the most current advice is to use `_asdict`.

Reply:
Seeing as calling `dict()` just runs `__dict__` anyway, that's certainly the far nicer approach, especially as it means we dont use private internal variables 👍

Reply:
neither of the above works because Token is just an object, so it can't be cast to a dict with `dict(marker)`, and it's not a `namedtuple`. I'm going to leave it as is, and have created an issue in https://github.com/sourcebots/sb-vision/issues/6 to make them `namedtuple`s.

Reply:
What about `var(marker)`?